src/Controller/HomeController.php

```diff
@@ -23,4 +23,9 @@ public function index()
     {
         return $this->twig->render('Home/index.html.twig');
     }
+    
+    public function seeDemands()
+    {
+        return $this->twig->render('Home/see_demands.html.twig');
+    }
```

Comment:
Having a /home/seeDemands URL is probably not a good choice because :
 - You should probably stick to lowercase URLs
 - It could be more understandable to have a /demand/list  (and maybe request is a better choice =)

And in fact this controller is not working because the Home/see_demands.html.twig file does not exists